### DISCUSSION TITLE
Bed Mesh Slots

### DISCUSSION THREAD

**Discussion by I-wish-2** [mriscoc/Ender3V2S1#772] [Q&A] [answered]
Greetings,
Does anyone know if the bed mesh slots, 0 1 2 etc. can have the full set of data in them deleted at one time?  Or do you have to edit all the positions in the mesh and set the z value to 0.00?
Thanks for the read
Doug

**Comment by mriscoc** [comment, marked as answer]
Try with:
```gcode
G29 P0  ; Set Mesh point to zero
G29 S0  ; Save Mesh in slot 0
G29 S1  ; Save Mesh in slot 1
...
```

**Reply by I-wish-2** [reply]
Thank you again Miguel.
Doug
